routing/cross_mwm_connector.cpp

```diff
@@ -54,7 +54,7 @@ bool CrossMwmConnector::IsTransition(Segment const & segment, bool isOutgoing) c
   if (transition.m_oneWay && !segment.IsForward())
     return false;
 
-  return (segment.IsForward() == transition.m_forwardIsEnter) == isOutgoing;
+  return (segment.IsForward() != transition.m_forwardIsEnter) == isOutgoing;
```

Comment:
@dobriy-eeh It's a fix in your code. Could you please recommend me a good name for value:
```segment.IsForward() != transition.m_forwardIsEnter```?

Probably it's better to rewrite this expression this way:
```(segment.IsForward() == transition.m_forwardIsEnter) != isOutgoing;```
And then name: ```(segment.IsForward() == transition.m_forwardIsEnter)``` as forwardIsEnter. And add some comment to explain the name? @dobriy-eeh what do you think?